Client/Frontend/Home/TopSites/TopSiteItemCell.swift

```diff
@@ -218,7 +249,13 @@ extension TopSiteItemCell: NotificationThemeable {
         titleLabel.textColor = UIColor.theme.homePanel.topSiteDomain
         faviconBG.backgroundColor = UIColor.theme.homePanel.shortcutBackground
         faviconBG.layer.shadowColor = UIColor.theme.homePanel.shortcutShadowColor
-        titleLabel.backgroundColor = UIColor.clear
+
+        let theme = BuiltinThemeName(rawValue: LegacyThemeManager.instance.current.name) ?? .normal
+        if theme == .dark {
+            sponsoredLabel.textColor = UIColor.Photon.LightGrey40
+        } else {
+            sponsoredLabel.textColor = UIColor.Photon.DarkGrey05
+        }
```

Comment:
nit: it could be a ternary condition
```suggestion
            sponsoredLabel.textColor = theme == .dark ? UIColor.Photon.LightGrey40 : UIColor.Photon.DarkGrey05
```